### ISSUE TITLE
http module, can it realize the ability to add header and autoChain?

### ISSUE THREAD

**Issue by GYiyouth** [labring/FastGPT#557]
**例行检查**

[//]: # '方框内填 x 表示打钩'

- [x] 我已确认目前没有类似 features
- [x] 我已确认我已升级到最新版本
- [x] 我已完整查看过项目 README，已确定现有版本无法满足需求
- [x] 我理解并愿意跟进此 features，协助测试和提供反馈
- [x] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 features 可能会被无视或直接关闭**

很不错的项目！但我阅读了文档，并经过本地部署试玩，发现有2个功能不知道如何开启，想咨询一下：
1：http模块，如何增加header请求呢？
2：如何实现类似autoChain能力呢？

**Comment by c121914yu** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
**Routine inspection**

[//]: # 'Fill x in the box to indicate tick'

- [x] I have confirmed that there are currently no similar features
- [x] I have confirmed that I have upgraded to the latest version
- [x] I have fully reviewed the project README and determined that the existing version cannot meet the needs.
- [x] I understand and am willing to follow up on this feature, assist with testing and provide feedback
- [x] I understand and agree with the above content, and understand that project maintainers have limited energy. **Features that do not follow the rules may be ignored or closed directly**

Very good project! But I read the documentation and tried it through local deployment, and found that there are 2 functions that I don’t know how to enable. I would like to ask:
1: http module, how to add header request?
2: How to achieve capabilities similar to autoChain?
